sentry_sdk/integrations/celery.py

```diff
@@ -46,10 +64,13 @@
 class CeleryIntegration(Integration):
     identifier = "celery"
 
-    def __init__(self, propagate_traces=True):
-        # type: (bool) -> None
+    def __init__(self, propagate_traces=True, celery_app=None):
```

Comment:
can't we auto discover the app?

Reply:
Hm, maybe I can get the app from the sender in the hook. I will have a look.

Reply:
Auto discovering works and look also better :-)